packages/client/src/client/muster.client.ts

```diff
@@ -24,4 +28,16 @@ export class MusterClient {
     });
   }
 
+  static getMusterComplianceByDate(orgId: number, unitName: string, query: GetMusterComplianceByDateQuery): Promise<GetMusterComplianceByDateResponse> {
```

Comment:
This function appears not to be used.

Reply:
I left this as a convenience method (so range didn't have to be used) but we can add to the chopping block in another PR